### PR TITLE
🔂 Fix repetition penalty

### PR DESCRIPTION
**Pull request by tengomucho**
# What does this PR do?

The generator was crashing when the request parameter container a repetition_penalty > 1.0. This was due to the logits processor expecting params to be torch tensors. 
So this is now fixed, and a test has been added to detect any possible regression.

### CODE REVIEW

**Comment by baptistecolle** [comment]
There are some failing tests 😕

**Review by baptistecolle — APPROVED** [review]
LGTM
